modules/builds-understanding-openshift-pipeline.adoc

```diff
@@ -14,20 +14,20 @@ Jenkins images on {product-title} are fully supported and users should follow Je
 
 Pipelines give you control over building, deploying, and promoting your applications on {product-title}. Using a combination of the Jenkins Pipeline build strategy, `jenkinsfiles`, and the {product-title} Domain Specific Language (DSL) provided by the Jenkins Client Plug-in, you can create advanced build, test, deploy, and promote pipelines for any scenario.
 
-*{product-title} Jenkins Sync Plugin*
+*{product-title} Jenkins Sync Plug-in*
 
-The {product-title} Jenkins Sync Plugin keeps the build configuration and build objects in sync with Jenkins jobs and builds, and provides the following:
+The {product-title} Jenkins Sync Plug-in keeps the build configuration and build objects in sync with Jenkins jobs and builds, and provides the following:
 
  * Dynamic job and run creation in Jenkins.
  * Dynamic creation of agent pod templates from image streams, image stream tags, or config maps.
  * Injecting of environment variables.
  * Pipeline visualization in the {product-title} web console.
- * Integration with the Jenkins git plugin, which passes commit information from
- * Synchronizing secrets into Jenkins credential entries {product-title} builds to the Jenkins git plugin.
+ * Integration with the Jenkins git plug-in, which passes commit information from
+ * Synchronizing secrets into Jenkins credential entries {product-title} builds to the Jenkins git plug-in.
 
-*{product-title} Jenkins Client Plugin*
+*{product-title} Jenkins Client Plug-in*
 
-The {product-title} Jenkins Client Plugin is a Jenkins plugin which aims to provide a readable, concise, comprehensive, and fluent Jenkins Pipeline syntax for rich interactions with an {product-title} API Server. The plugin uses the {product-title} command line tool, `oc`, which must be available on the nodes executing the script.
+The {product-title} Jenkins Client Plug-in is a Jenkins plug-in which aims to provide a readable, concise, comprehensive, and fluent Jenkins Pipeline syntax for rich interactions with an {product-title} API Server. The plug-in uses the {product-title} command line tool, `oc`, which must be available on the nodes executing the script.
 
 The Jenkins Client Plug-in must be installed on your Jenkins master so the {product-title} DSL will be available to use within the `jenkinsfile` for your application. This plug-in is installed and enabled by default when using the {product-title} Jenkins image.
```

Comment:
This is not part of the PR but to be consistent can you change the Plug-in to Plugin, please?